### ISSUE TITLE
Rewrite navigation to use routing

### ISSUE THREAD

**Issue by Derstilon** [yaptide/ui#1034]
This will allow us to immediately open the simulations or results tab when opening examples from URL.

- [ ] Add routes for all tabs in the navigation
- [ ] handle from any route loading files from URL passed as a link parameter

**Comment by grzanka** [comment]
Good idea, can help to handle loading examples and pointing user directly to results
